target/linux/ipq40xx/config-5.4

```diff
@@ -536,3 +536,4 @@ CONFIG_ZLIB_DEFLATE=y
 CONFIG_ZLIB_INFLATE=y
 CONFIG_ZSTD_COMPRESS=y
 CONFIG_ZSTD_DECOMPRESS=y
+CONFIG_IPQ40XX_PORT_ISOLATION=n
```

Comment:
```
# CONFIG_IPQ40XX_PORT_ISOLATION is not set
```

Reply:
Kernel config can only be set per (sub)target, not per device.

Reply:
And there is no other way of doing something like this?

Reply:
@blocktrron I changed this to y